src/main/java/io/github/debuggyteam/architecture_extensions/util/SafeRenderLayer.java

```diff
@@ -1,13 +1,33 @@
 package io.github.debuggyteam.architecture_extensions.util;
 
-public enum SafeRenderLayer {
-	SOLID,
-	TRANSLUCENT,
-	CUTOUT;
-
-	public static SafeRenderLayer combine(SafeRenderLayer first, SafeRenderLayer second) {
-		if (first == TRANSLUCENT || second == TRANSLUCENT) return TRANSLUCENT;
-		if (first == CUTOUT || second == CUTOUT) return CUTOUT;
-		return SOLID;
+import java.util.function.Supplier;
+
+import org.quiltmc.loader.api.minecraft.ClientOnly;
+
+import net.minecraft.client.render.RenderLayer;
+
+public enum SafeRenderLayer implements @ClientOnly Supplier<RenderLayer> {
+	SOLID(0),
+	TRANSLUCENT(2),
+	CUTOUT(1);
+
+	private final int priority;
+
+	SafeRenderLayer(int priority) {
+		this.priority = priority;
+	}
+
+	@Override
+	public RenderLayer get() {
```

Comment:
It's shocking that this runs on dedicated server at all. Can we restrict this method to the client environment only?

Reply:
It is

Reply:
Also thank you for *testing* on a dedicated server. It clearly runs, I just want to stay way on the safe side.

Reply:
I will put it a client only annotation on it anyways, so people can clearly see it's for use on the client *only*